docs/DeveloperGuide.md

```diff
@@ -694,7 +697,9 @@ Those without any stars are user stories that were considered but will not be im
 
 
 
-**Use case: UCSH06 Sort contacts in alphabetical order**
+
+### **Use case: UCSH10 — Sort contacts in alphabetical order**
```

Comment:
Same for this

Reply:
This was in the original DeveloperGuide. I also thought of deleting it but wasn't sure as it was part of our Use Cases google sheets